Remove explicit mips64 -mabi parameter

This breaks when building for n32 ABI on mips64.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
[Bernd: rebased for botan-2.7.0]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---

diff -Nuar Botan-1.10.8-orig/src/build-data/cc/gcc.txt Botan-1.10.8/src/build-data/cc/gcc.txt
--- Botan-1.10.8-orig/src/build-data/cc/gcc.txt	2014-04-10 17:11:44.000000000 +0300
+++ Botan-1.10.8/src/build-data/cc/gcc.txt	2014-10-23 09:20:34.506722323 +0300
@@ -83,7 +83,6 @@
 
 openmp  -> "-fopenmp"
 
-mips64  -> "-mabi=64"
 s390    -> "-m31"
 s390x   -> "-m64"
 sparc32 -> "-m32 -mno-app-regs"
